usdigitalresponse/usdr-gost
# feat: adds endpoint to grab grants

### Ticket #1745 
## Description
- Adds ability to retrieve grants with the new filters added by the new search experience

## Screenshots / Demo Video
https://www.loom.com/share/081fbbc45a71491bbacd52f282ad28dc

## Testing
N/A - Will have a follow-up PR

### Automated and Unit Tests

### Manual tests for Reviewer
Seed the local database with grants
Pull down the branch
Create and modify filters to see the new search results


## Checklist
